### ISSUE TITLE
Multiple subscription add sample rest API request is incorrect

### ISSUE THREAD

**Issue by deelaka96** [wso2/docs-apim#1045]
**Description:**
In the documentation [1] to add multiple subscriptions to an API, the provided sample request is incorrect.

```
POST https://localhost:9443/api/am/store/v0.14/subscriptions/multiple
Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8

{
    "tier": "Gold",
    "apiIdentifier": "c43a325c-260b-4302-81cb-768eafaa3aed",
    "applicationId": "c30f3a6e-ffa4-4ae7-afce-224d1f820524"
}
```
The request payload should be changed as follows ( as an array of JSONs ),
```
POST https://localhost:9443/api/am/store/v0.14/subscriptions/multiple
Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8

[{
    "tier": "Gold",
    "apiIdentifier": "c43a325c-260b-4302-81cb-768eafaa3aed",
    "applicationId": "c30f3a6e-ffa4-4ae7-afce-224d1f820524"
}]
```
**Suggested Assignees:**
@Mariangela 

**Affected Product Version:**
APIM 2.6.0

[1]https://docs.wso2.com/display/AM260/apidocs/store/index.html#!/operations#SubscriptionMultitple#subscriptionsMultiplePost

**Comment by msm1992** [comment]
This issue is fixed in 3.x docspaces, this needs to be fixed in 2.x only.

**Comment by Mariangela** [comment]
@Chamindu36 , please fix this in the 2.6.0 docs.
